target/linux/ar71xx/image/Makefile

```diff
@@ -18,6 +18,7 @@ include ./generic-legacy-devices.mk
 include ./generic-senao.mk
 include ./generic-tp-link.mk
 include ./generic-ubnt.mk
+include ./atheros.mk
```

Comment:
Pretty sure this is "wrong", no sure about the proper solution however

Reply:
Do it like everone else does it. Check target/linux/ar71xx/image/generic.mk